lib/src/component_declaration/annotations.dart

```diff
@@ -15,6 +15,8 @@
 // Dummy annotations that would be used by Pub code generator
 library over_react.component_declaration.annotations;
 
+export 'package:meta/meta.dart' show mustCallSuper;
```

Comment:
Oh interesting, this is so that generated code can use `@mustCallSuper`, right? Could we add a comment here to that effect.

Kind of gross that it has to be exported from over_react and added to its public API, though I don't think it would cause any issues... 

And I can't think of any better alternatives, unless we created an alias for it that was named something else, and maybe deprecated (e.g., `@Deprecated('For generated use only) const $overReact$mustCallSuper = mustCallSuper;`)... Not sure if that's worth the trouble though.